.github/workflows/PR.yml

```diff
@@ -0,0 +1,46 @@
+
+name: Libpalaso CI workflow
+
+on:
+  pull_request:
+    branches: [ feature/nuget ]
+
+jobs:
+
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true #this is needed for Setup MSBuild.exe 
+      CI: GithubActions
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```

Comment:
For security reasons it would be better to use the commit sha1 instead of the tag - a git tag could be changed by a malicious user whereas commits are immutable.